### PR TITLE
Update for Scala 3.3.3 and 2.12.19

### PR DESCRIPTION
**Pull request by mtomko**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,11 +2,11 @@ inThisBuild(Seq(
   organization := "com.github.ghik",
   scalaVersion := crossScalaVersions.value.head,
   crossScalaVersions := Seq(
-    "3.4.1-RC1", "3.4.0",
-    "3.3.2", "3.3.1", "3.3.1-RC7", "3.3.1-RC6", "3.3.1-RC5", "3.3.1-RC4", "3.3.1-RC3", "3.3.1-RC2", "3.3.1-RC1", "3.3.0",
+    "3.4.0",
+    "3.3.3", "3.3.1", "3.3.1-RC7", "3.3.1-RC6", "3.3.1-RC5", "3.3.1-RC4", "3.3.1-RC3", "3.3.1-RC2", "3.3.1-RC1", "3.3.0",
```
Comment (ghik) [inline build.sbt]: Why was 3.3.2 removed?

Comment (mtomko) [reply build.sbt]: It was never released. https://www.scala-lang.org/blog/2024/02/29/scala-3.4.0-and-3.3.3-released.html

Comment (mtomko) [reply build.sbt]: I do see that they maybe released a 2.12.19, I can add that too.

Comment (mtomko) [reply build.sbt]: Oh, weird, actually something is wrong here. I didn't intentionally remove 3.3.2, I thought I just added 3.4.0. Let me re-do this.

Comment (mtomko) [reply build.sbt]: Sorry, I updated my local fork but my local repo must have been missing changes.

**Comment by mtomko** [comment]
Sorry this is too messed up. I'll try again from fresh, or let someone else do it.

**Pull request closed without merging**